test/e2e/libpod_suite_test.go

```diff
@@ -129,12 +132,16 @@ func PodmanTestCreate(tempDir string) *PodmanTestIntegration {
 		cgroupManager = "cgroupfs"
 	}
 
-	runCBinary, err := exec.LookPath("runc")
-	// If we cannot find the runc binary, setting to something static as we have no way
-	// to return an error.  The tests will fail and point out that the runc binary could
-	// not be found nicely.
-	if err != nil {
-		runCBinary = "/usr/bin/runc"
+	if os.Getenv("RUNC_BINARY") != "" {
```

Comment:
If we are going to do this it should be PODMAN_RUNC_BINARY, or LIBPOD_RUNC_BINARY

Reply:
Never mind this is for tests.